crates/starknet-os/src/execution/deprecated_syscall_handler.rs

```diff
@@ -91,7 +91,7 @@ where
     pub async fn call_contract(&self, syscall_ptr: Relocatable, vm: &mut VirtualMachine) -> Result<(), HintError> {
         self.call_contract_and_write_response(syscall_ptr, CallContract::response_offset(), vm).await
     }
-    #[allow(unused)]
```

Comment:
This warning that we silenced here might have tipped us off a long time ago that there was a problem :)